### PR TITLE
Updated the dodge_the_creeps demo to include the "Removing the creeps" part in the tutorial.

### PR DESCRIPTION
**Pull request by EludeQ**
This is for #454.
A [small part from the tutorial](https://docs.godotengine.org/en/3.2/getting_started/step_by_step/your_first_game.html#removing-old-creeps) for the dodge_the_creeps demo was missing in the project here.

### CODE REVIEW

File: 2d/dodge_the_creeps/Mob.gd [hunk 1]
```diff
@@ -11,3 +11,6 @@ func _ready():
 
 func _on_VisibilityNotifier2D_screen_exited():
 	queue_free()
+
+func _on_start_game():
```
Comment (aaronfranke) [inline 2d/dodge_the_creeps/Mob.gd:15]: There should be two lines between methods to follow the [GDScript style guide](https://docs.godotengine.org/en/latest/getting_started/scripting/gdscript/gdscript_styleguide.html).

File: 2d/dodge_the_creeps/Main.gd [hunk 2]
```diff
@@ -33,6 +33,7 @@ func _on_MobTimer_timeout():
 	direction += rand_range(-PI / 4, PI / 4)
 	mob.rotation = direction
 	mob.linear_velocity = Vector2(rand_range(mob.min_speed, mob.max_speed), 0).rotated(direction)
+	$HUD.connect("start_game", mob, "_on_start_game")
```
Comment (aaronfranke) [inline 2d/dodge_the_creeps/Main.gd:36]: The function 'connect()' returns a value, but this value is never used.

Please add `# warning-ignore:return_value_discarded` above this line to silence the warning.

Comment (EludeQ) [reply 2d/dodge_the_creeps/Main.gd:36]: Alright! I've added the warning ignore and added that missing line. I've also added this to the C# version.

File: mono/dodge_the_creeps/Main.cs [hunk 3]
```diff
@@ -3,86 +3,88 @@
 
 public class Main : Node
 {
-    [Export]
-    public PackedScene mob;
+	[Export]
+	public PackedScene mob;
```
Comment (aaronfranke) [inline mono/dodge_the_creeps/Main.cs:7]: Because it's the C# standard, we use spaces for indentation exclusively in C# files.

Comment (EludeQ) [reply mono/dodge_the_creeps/Main.cs]: Woops! Didn't know Godot changes indents automatically. Should be fixed now.

File: 2d/dodge_the_creeps/Main.gd (hunk 2 above)
Comment (aaronfranke) [reply 2d/dodge_the_creeps/Main.gd:36]: But you didn't actually add this line.

**Pull request merged**